### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by christophemassin** [WeblateOrg/weblate#1379]
### Steps to reproduce
1. Try to get connect to the app after installation 
2. got Internal Server Error message
3.

### Actual behaviour
Internal server error when trying to reach the website 

Tue Feb 21 13:24:44 2017 - your mercy for graceful operations on workers is 60 seconds
Tue Feb 21 13:24:44 2017 - mapped 230592 bytes (225 KB) for 2 cores
Tue Feb 21 13:24:44 2017 - *** Operational MODE: preforking ***
Tue Feb 21 13:24:44 2017 - added /var/www/weblate/ to pythonpath.
Traceback (most recent call last):
  File "/var/www/weblate/weblate/wsgi.py", line 44, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/dist-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/dist-packages/django/__init__.py", line 22, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 53, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 97, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/var/www/weblate/weblate/__init__.py", line 104, in <module>
    check_requirements()
  File "/var/www/weblate/weblate/requirements.py", line 312, in check_requirements
    versions = get_versions() + get_optional_versions()
  File "/var/www/weblate/weblate/requirements.py", line 198, in get_versions
    '1.2.0',
  File "/var/www/weblate/weblate/requirements.py", line 152, in get_single
    mod = get_version_module(module, name, url)
  File "/var/www/weblate/weblate/requirements.py", line 48, in get_version_module
    url,
Exception: Failed to import social_core, please install social-auth-core from http://python-social-auth.readthedocs.io/
Tue Feb 21 13:24:44 2017 - unable to load app 0 (mountpoint='') (callable not found or import error)
Tue Feb 21 13:24:44 2017 - *** no app loaded. going in full dynamic mode ***
Tue Feb 21 13:24:44 2017 - *** uWSGI is running in multiple interpreter mode ***
Tue Feb 21 13:24:44 2017 - spawned uWSGI master process (pid: 13930)
Tue Feb 21 13:24:44 2017 - spawned uWSGI worker 1 (pid: 13945, cores: 1)
Tue Feb 21 13:24:44 2017 - spawned uWSGI worker 2 (pid: 13946, cores: 1)

I think there is some mismatch between the install and the server about the package ? how can i force to use python env ? ( if it's the good one to use ? )

### Expected behaviour
Tell us what should happen

### Server configuration
/manage.py list_versions
 * Weblate weblate-2.11-131-g7d2c05f
 * Python 2.7.12
 * Django 1.10
 * six 1.10.0
 * social-auth-core 1.2.0
 * social-auth-app-django 1.1.0
 * django-appconf 1.0.1
 * Translate Toolkit 2.0.0
 * Whoosh 2.7.0
 * defusedxml 0.4.1
 * Git 2.7.4
 * Pillow (PIL) 1.1.7
 * dateutil 2.4.2
 * lxml 3.5.0
 * django-crispy-forms 1.6.0
 * compressor 2.1.1
 * djangorestframework 3.4.0
 * pytz 2014.10
 * pyLibravatar N/A
 * PyYAML 3.11
 * Mercurial 3.7.3
 * Database backends: django.db.backends.mysql

However, i ve set it up in /var/www with group and user www-data . But, i have to sudo the command to be able to run it . What are the proper user / right to have on the repository

**Comment by christophemassin** [comment]
cause i got already : 

pip install social-auth-core
Collecting social-auth-core
  Using cached social_auth_core-1.2.0-py2-none-any.whl
Collecting requests>=2.9.1 (from social-auth-core)
  Using cached requests-2.13.0-py2.py3-none-any.whl
Collecting requests-oauthlib>=0.6.1 (from social-auth-core)
  Using cached requests_oauthlib-0.8.0-py2.py3-none-any.whl
Collecting python-openid>=2.2.5 (from social-auth-core)
Collecting PyJWT>=1.4.0 (from social-auth-core)
  Using cached PyJWT-1.4.2-py2.py3-none-any.whl
Collecting oauthlib>=1.0.3 (from social-auth-core)
Collecting six>=1.10.0 (from social-auth-core)
  Using cached six-1.10.0-py2.py3-none-any.whl
Installing collected packages: requests, oauthlib, requests-oauthlib, python-openid, PyJWT, six, social-auth-core
Successfully installed PyJWT-1.3.0 oauthlib-1.0.3 python-openid-2.2.5 requests-2.9.1 requests-oauthlib-0.4.0 six-1.10.0 social-auth-core-1.2.0

**Comment by nijel** [comment]
Have you looked at our [installation instructions](https://docs.weblate.org/en/latest/admin/install.html)?

**Comment by christophemassin** [comment]
Hello, 

Yes, and i have already scratch a server once cause it was not working, before i start opening issue. I follow all the step of the install but encounter issue after issue. I m not familiar with python, so there is perhaps some notion i don't have / need to be enhance in the step. If i got time when it will be working, i will automatise the install script cause it's hard to follow when you're not familiar with it

**Comment by christophemassin** [comment]
I drop it and used the amazon marketplace instance instead
